src/controls/Mobile.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react'
+import { useStore } from '../store'
+
+const pressed = []
+
+function useTouch(target, event, up = true) {
+  useEffect(() => {
+    const downHandler = (e) => {
+      if (target.indexOf(e.target.value) !== -1) {
+        const isRepeating = !!pressed[e.target.value]
```

Comment:
The value of this array is already a `boolean` or `null` no need to have the `!!` in check as in the `if` statement the browser will already check it for truthiness.

Reply:
The controls have changed since this PR has been opened

Reply:
Have all of the controls changed or specific controls that can get updated?
please list them.

How can we set this pr up for success? @njm222 , thanks

Reply:
From what I remember this is good to go, it just needs to be merged with main and resolve conflicts. 

I would rebase onto main or force push a new branch since a lot has changed since this PR was opened.